### ISSUE TITLE
Clean up Dependencies

### ISSUE THREAD

**Issue by arb** [continuationlabs/hapi-setup#33]
Finalize dev vs run dependencies. Lock down versions where appropriate.

**Comment by AdrieanKhisbe** [comment]
@arb, Do you want me to take care of this in #34? :)

**Comment by arb** [comment]
Nah, this can be it's own thing. Really we want to be clear on dev vs actual dependencies. Since this module has a build step, it's not immediately obvious which dependencies you need where.
